api/services/handle_event_service.py

```diff
@@ -3,15 +3,21 @@
 from api import models, db, app
 from . import registration_service as registration
 from . import call_log_event_service as call_log_event
+from utils.loggingutils import logger
+import traceback
 
 
 class HandleEventService:
     def handle_event_service(self, form_data):
+        logger.info("Processing call event for call sid %s", form_data["CallSid"])

```

Comment:
Lets not add this log as it will be printed for every webhook call

Reply:
Updated